### PR TITLE
Fixed grammer usage of parole

### PR DESCRIPTION
**Pull request by tmbutterworth**
Parole was being used in the wrong direction asking the navy to provide it instead of requiring them to accept the conditions to receive it.

### CODE REVIEW

**Comment by Nescio0** [comment]
There is no need to close requests and open new ones on the same issue. It's perfectly possible to edit files or add new changes.

As for "parole", there is a difference between the mass noun "the temporary or permanent release of a prisoner before the expiry of a sentence, on the promise of good behaviour" and the count noun "a promise or undertaking given by a prisoner of war to return to custody or act as a non-belligerent if released". (And there is also the verb "to parole": "release (a prisoner) on parole".)
In other words, prisoners can give parole (i.e. a formal promise) and can be paroled (be released on parole).

**Comment by tmbutterworth** [comment]
> There is no need to close requests and open new ones on the same issue. It's perfectly possible to edit files or add new changes.
> 
> As for "parole", there is a difference between the mass noun "the temporary or permanent release of a prisoner before the expiry of a sentence, on the promise of good behaviour" and the count noun "a promise or undertaking given by a prisoner of war to return to custody or act as a non-belligerent if released". (And there is also the verb "to parole": "release (a prisoner) on parole".)
> In other words, prisoners can give parole (i.e. a formal promise) and can be paroled (be released on parole).

Prisoners or war, like prisoners in penal institutions receive parole from their captor upon meeting required conditions they do not give parole to the captor. Prisoners do not dictate parole to their captors as they have no power or authority. In the US military armed forces members are not allowed to accept parole from a captor because it constitutes serving a foreign government.

I know parole has lots of definitions but the current wording sounds like they are asking the navy prisoners to give them parole even though they are the captors. In military and prison only the group with authority to create the conditions can "give parole".

Mixing other rarely used definitions just creates confusion and most people just treat it as a typo.

**Comment by tmbutterworth** [comment]
> There is no need to close requests and open new ones on the same issue. It's perfectly possible to edit files or add new changes.

Thanks I just figured out how to do that.

**Comment by comnom** [comment]
>I know parole has lots of definitions but the current wording sounds like they are asking the navy prisoners to give them parole even though they are the captors.

Keep in mind that we're talking about the people who will become the Oathkeepers here.

So not only is the change not correcting any errors, it's weakening the statement from "a promise confirmed by a pledge of one's honor", to just any old joe giving his word.

FYI, the same usage is present in other missions concerning the soon to be Oathkeepers.

**Comment by tmbutterworth** [comment]
> So not only is the change not correcting any errors, it's weakening the statement from "a promise confirmed by a pledge of one's honor", to just any old joe giving his word.

Pledging their honor and not to take up arms against the free worlds again is the conditional release to receive parole. Parole is being given by the free worlds to the navy. Parole of prisoners is given by the captor to the prisoner not the prisoner to the captor. It doesn't weaken it, it places it properly in common understanding. Not many people know parole as a "Secret Word" given to naval officers, which is the only way I see it could work as something the navy could give.

**Comment by Nescio0** [comment]
Parole literally means a formal promise. The prisoners have to give their word to the authorities (as is clear from the current sentence), not the other way around (as your proposal suggests).

**Comment by tmbutterworth** [comment]
> Parole literally means a formal promise. The prisoners have to give their word to the authorities (as is clear from the current sentence), not the other way around (as your proposal suggests).

If the navy won the battle and took free world captives parole would be theirs to give.

Any Navy members who refuse to swear to the conditions clearly would not receive parole as parole is a state of being/living condition. Prisoners or war in the dark ages that accepted parole then rejoined their unit to fight were clearly put to death if recaptured.

Only the side with the power to dictate the conditions can give parole the captive can only accept or decline it. A prisoner does no give parole to th e prison they receive it if the agree to the conditions being forced on them.
